docker/.env.example

```
@@ -0,0 +1,139 @@
+############
+# Secrets
+# YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+############
+
+POSTGRES_PASSWORD=your-super-secret-and-long-postgres-password
+JWT_SECRET=your-super-secret-jwt-token-with-at-least-32-characters-long
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the POSTGRES_PASSWORD key

(UnorderedKey)